android/app/src/main/AndroidManifest.xml

```diff
@@ -75,5 +75,9 @@
             android:name=".AudioService"
             android:enabled="true"
             android:foregroundServiceType="mediaPlayback" />
+
+        <service
+            android:name="id.flutter.flutter_background_service.BackgroundService"
```

Comment:
This task of background service might run into some issues on some models of phone because of battery optimizations, possibly something to do with battery optimizations https://stackoverflow.com/questions/39256501/check-if-battery-optimization-is-enabled-or-not-for-an-app/54982071#54982071

Reply:
Thinking about this and how the beacon app is maybe we should request it be turned off before the tts service starts

Reply:
The service doesn't run unless explicitly requested and foregrounded.

Reply:
Ok then this isn't necessary

Reply:
No, the service declaration is still necessary otherwise the server won't be able to start. The manifest doesn't cause the service to run.

Reply:
I meant the battery optimization bit which I was worried about.